ui/src/app/features/app-user-list/application-user-list.component.ts

```diff
@@ -125,20 +125,20 @@ export class ApplicationUserListComponent implements OnInit, OnDestroy {
 
     const dialogSubscription = dialog.afterClosed().subscribe(({ userEmail, role }) => {
       if (userEmail && role) {
-        this.appUserListFacade.inviteUser(userEmail, role).subscribe(() => this.openEmailNotification(userEmail));
+        this.appUserListFacade.inviteUser(userEmail, role).subscribe(
```

Comment:
please refactor to dispatch an action. Handle response and display notifications in effects.